evalml/automl/engine/sequential_engine.py

```diff
@@ -34,7 +34,7 @@ def evaluate_batch(self, pipelines):
             X, y = self.X_train, self.y_train
             if pipeline.model_family == ModelFamily.ENSEMBLE:
                 X, y = self.X_train.iloc[self.ensembling_indices], self.y_train.iloc[self.ensembling_indices]
-            elif self.ensembling_indices is not None:
+            elif self.ensembling_indices is not None:   # Is this necessary?
```

Comment:
Someone will have to comment on this.  When this got extracted from AutoML into the engine, we lost track of who put it in there.  It's kind of strange, undocumented behavior.